### PR TITLE
Jr index.html

### PR DESCRIPTION
**Pull request by jessrod11**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,6 +10,28 @@
...
+    <button id="stix">Stix</button>
+    <button id="anna">Anna</button>
+    <button id="mary-alice">Mary-Alice</button>
+    <button id="ana">Ana</button>
+  </div>
+
+  <div id="main-container">
+    <div id="message-container"></div>
+    <div id="theme-container">
+      <input type="radio" id="night-sky"> Night Sky
```
Comment (ghost) [inline index.html]: Can we put closing </input> tag

File: index.html [hunk 2]
```diff
@@ -10,6 +10,28 @@
...
+      <input type="radio" id="rainbow-text"> Rainbow Text
+      <input type="radio" id="larger-text"> Large Text
+    </div>
+  </div>
+
+  <div id="footer-container">
+    <button type="button" class="btn btn-danger">Clear All</button>
+    <input aria-placeholder="Enter Message">
```
Comment (ghost) [inline index.html]: and here as well

**Comment by AnastasiiaSomchyk** [comment]
👍

**Comment by ghost** [comment]
👍

**Comment by maryaliceorr** [comment]
👍

**Pull request merged**